### PR TITLE
Chrome 5/44 added `webextensions.api.history` support

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects version values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) for the `history` Web Extensions interface. The data comes from a commit in the browser's source code, mapped to a version number using available tooling or via the commit timestamp.

Commits:

- 5: https://source.chromium.org/chromium/chromium/src/+/bb0a6a0af3be6d24a5c8f9ebd2e686347a5c882d
- 44: https://source.chromium.org/chromium/chromium/src/+/00f1fc22bfcc653b47d62778170af3eee9855720

### CODE REVIEW

File: webextensions/api/history.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/addUrl",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:115]: :x: https://github.com/chromium/chromium/commit/bb0a6a0af3be6d24a5c8f9ebd2e686347a5c882d in [5.0.368.0](https://github.com/chromium/chromium/releases/tag/5.0.368.0) before 5.0.375.

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 2]
```diff
@@ -197,7 +197,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/deleteAll",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:200]: :x: Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 3]
```diff
@@ -219,7 +219,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/deleteRange",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:222]: :x: Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 4]
```diff
@@ -241,7 +241,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/deleteUrl",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:244]: :x: Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 5]
```diff
@@ -263,7 +263,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/getVisits",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:266]: :x: Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 6]
```diff
@@ -329,7 +329,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/onVisited",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:332]: :x: Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 7]
```diff
@@ -352,7 +352,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/onVisitRemoved",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:355]: :x: Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 8]
```diff
@@ -374,7 +374,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/search",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:377]: :x: Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 9]
```diff
@@ -27,7 +27,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/HistoryItem",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:30]: :x: Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 10]
```diff
@@ -68,7 +68,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/TransitionType",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "44"
```
Comment (caugner) [inline webextensions/api/history.json:71]: ✅ https://github.com/chromium/chromium/commit/00f1fc22bfcc653b47d62778170af3eee9855720 in [44.0.2359.0](https://github.com/chromium/chromium/releases/tag/44.0.2359.0) before 44.0.2403.

File: webextensions/api/history.json [hunk 11]
```diff
@@ -90,7 +90,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history/VisitItem",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:93]: ❌ Like `addUrl`:

```suggestion
                "version_added": "5"
```

File: webextensions/api/history.json [hunk 12]
```diff
@@ -47,7 +47,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤58"
+                  "version_added": "6"
```
Comment (caugner) [inline webextensions/api/history.json:50]: :x: Like `addUrl`:

```suggestion
                  "version_added": "5"
```

**Review by caugner — CHANGES_REQUESTED** [review]

File: webextensions/api/history.json [hunk 13]
```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history",
           "support": {
             "chrome": {
-              "version_added": "≤58"
+              "version_added": "6"
```
Comment (queengooborg) [inline webextensions/api/history.json:9]:
```suggestion
              "version_added": "5"
```

**Review by caugner — APPROVED** [review]

**Pull request merged**